devenv-tasks/src/lib.rs

```diff
@@ -81,10 +81,25 @@ pub struct TaskConfig {
     inputs: Option<serde_json::Value>,
 }
 
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, Deserialize, Serialize, clap::ValueEnum)]
+#[serde(rename_all = "snake_case")]
+pub enum RunMode {
+    #[default]
+    /// Run only the specified task without dependencies
+    Single,
+    /// Run the specified task and all tasks that depend on it (downstream tasks)
+    After,
+    /// Run all dependency tasks first, then the specified task (upstream tasks)
```

Comment:
This should have been the default

Reply:
Yeah, maybe in the context of tasks. And it should be single for the CLI.